io.anytype.anytype.yml

```diff
@@ -0,0 +1,45 @@
+id: io.anytype.anytype
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+command: anytype
+finish-args:
+  - --device=dri
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --talk-name=org.freedesktop.secrets
+  - --env=ELECTRON_OZONE_PLATFORM_HINT=wayland
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - name: anytype
+    buildsystem: simple
+    build-commands:
+      - ar -o -x anytype_amd64.deb
+      - tar --extract --file data.tar.xz
+      - cp --preserve --recursive opt/* usr/* /app
+      - desktop-file-edit --set-key Exec --set-value "/app/bin/anytype %U" /app/share/applications/anytype.desktop
+      - desktop-file-edit --set-key Icon --set-value "io.anytype.anytype" /app/share/applications/anytype.desktop
```

Comment:
`rename-icon` use like `rename-desktop-file` should do the trick.

Reply:
Im sorry, I don't understand what you mean, can you explain?

Reply:
`rename-icon` that you should use to use the icons provided by the binary pacakge WILL update the .desktop file, making this unecessary. Works like `rename-desktop-file`

Reply:
Okay, removed the svg and renamed the icons using `rename-icon`.